db/migrate/20160712043931_create_storage_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateStorageProfiles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :storage_profiles do |t|
+      t.string :name
+      t.string :uuid
```

Comment:
Should this be `ems_ref`?  It happens to be a UUID on vSphere but if other providers add a storage profile concept they might not use UUIDs